vendor/utils/list.mli

```diff
@@ -0,0 +1,7 @@
+val partition_map :
+  'a list -> f:('a -> [`Fst of 'b | `Snd of 'c]) -> 'b list * 'c list
```

Comment:
This is using the base 0.13 API. What do you think about using the new API (based on `Either`) and updating the call sites so that this function is easier to delete when we move the support window?

Reply:
done